modules/nf-core/fastp/meta.yml

```diff
@@ -27,6 +27,9 @@ input:
       type: file
       description: File in FASTA format containing possible adapters to remove.
       pattern: "*.{fasta,fna,fas,fa}"
+  - save_trimmed_pass:
+      type: boolean
+      description: Specify true to save files that pass trimming thresholds, ending in `*.fastp.fastq.gz`
```

Comment:
I am with James, but it's not a hill-to-die-on for me. However, if we go for this logic, I would like to have a big warning here, that it is not just about saving, which for me somewhat implies it is about the results directory only, BUT also to get any read output from the module.

Reply:
Maybe allow by default, and have a `disable_save_trimmed_pass` instead?

Reply:
I think everyone would rather have something instead of nothing